README.md

```diff
@@ -283,6 +283,19 @@ eventBus.publish({type: 'MOVE_CARD', fromLaneId: 'PLANNED', toLaneId: 'WIP', car
 //To update the lanes
 eventBus.publish({type: 'UPDATE_LANES', lanes: newLaneData})
 
+//To update one lane
+eventBus.publish({type: 'UPDATE_LANE', lane: { id: 'TODO', title: 'teste' }})
+
+//To add a new lane
+eventBus.publish({type: 'ADD_LANE', lane: { id: 'TODO', title: 'teste' }})
+
+//To move one lane
+eventBus.publish({type: 'MOVE_LANE', fromIndex: 0, toIndex: 1 })
```

Comment:
I'm asking myself if was not better to create a new action to operate based on id, and no index. What do you think @dapi?

Reply:
Hi @LucasBadico !

I agree with you. If we pass an id we can fetch lane's index when need.

What is you use case of `MOVE_LANE` event? Do you save lanes order to backend?

One thought came. When we drug one lane, from example move it from index 0 to index 5. Actually all lanes from 1 to 5 index are moved ) Shall we emit events for all these lanes? Probably better way to emit event like "LANES_MOVED" and pass new ordered lanes set?

Reply:
You have some points.
but my use case here:
- move lane was needed because the WebSocket and the feature of move lane and the socket keep the order of all clients synched.
- would be a good thing to have that Lanes Moved, but now I'm doing this on my backend, with a head and tail logic.